### PR TITLE
XS-3229: Update 0005 and 0015 from next_q1_16 to master

### PR DESCRIPTION
**Pull request by chrislococo-wf**
### Changes:

- patched dqc_us_0005.py to the latest in next_q1_16 for immediate release
- patched dqc_us_0015.py to the latest in next_q1_16 for immediate release
- updated version pin from 1.0.0 to 1.1.0 for release.
- patched the test files for 0015 and 0005.
- brought over the util/neg_num.py file as it is used by 0015.

### Pull request process

- [x] Full description of changes provided above.
- [x] Reviewed by another person. (+1)
- [x] If this is a code change, reviewed by a second person. (+1)
- [x] QA'd to the specifications listed [here](https://github.com/DataQualityCommittee/dqc_us_rules#quality-assurance-qa-of-a-pull-request). Documentation only changes do not require the formal code quality assurance process. (+10)

### CODE REVIEW

File: dqc_us_rules/dqc_us_0015.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-import csv
 import os
-from .util import facts, messages
+import csv
+from .util import messages, neg_num
+from .util import facts as facts_util
 
 _CODE_NAME = 'DQC.US.0015'
-_RULE_VERSION = '1.0'
+_RULE_VERSION = '2.0.0'
```
Comment (jasonleinbach-wf) [inline dqc_us_rules/dqc_us_0015.py]: These aren't 2.0 on master yet.

File: dqc_us_rules/dqc_us_0005.py [hunk 2]
```diff
@@ -13,6 +13,8 @@
 _CODE_NAME = 'DQC.US.0005'
 _RULE_VERSION = '1.0'
```
Comment (jasonleinbach-wf) [inline dqc_us_rules/dqc_us_0005.py]: This needs an update too.

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by sagesmith-wf** [comment]
+1

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by sagesmith-wf** [comment]
+1

**Comment by jasonleinbach-wf** [comment]
+10 - verified docs from the following tickets:
XS-2308, XS-3078, XS-3037, XS-3079
All fired as expected, no regressions.

**Comment by jasonleinbach-wf** [comment]
Reviewed, verified.  Merging.

**Pull request merged**